crypto/monero/address.go

```diff
@@ -11,26 +12,24 @@ import (
 const (
 	addressPrefixMainnet  byte = 18
 	addressPrefixStagenet byte = 24
-
-	// AddressLength is the length of a Monero address
-	AddressLength = 1 + 32 + 32 + 4
 )
 
 var (
-	errChecksumMismatch         = fmt.Errorf("invalid address checksum")
-	errInvalidAddressLength     = fmt.Errorf("invalid monero address length")
-	errInvalidPrefixGotMainnet  = fmt.Errorf("invalid monero address: expected stagenet, got mainnet")
-	errInvalidPrefixGotStagenet = fmt.Errorf("invalid monero address: expected mainnet, got stagenet")
+	errChecksumMismatch         = errors.New("invalid address checksum")
+	errInvalidAddressLength     = errors.New("invalid monero address length")
+	errInvalidAddressEncoding   = errors.New("invalid monero address encoding")
```

Comment:
I was adding `errInvalidAddressEncoding` and didn't want to use `fmt.Errorf` as there was no string formatting, so I just updated them all to match.